openstack/nova/values.yaml

```diff
@@ -1013,3 +1013,84 @@ vpa:
 
 # used to set the PYTHONWARNINGS environment variable everywhere
 python_warnings: "ignore:Unverified HTTPS request,ignore::SyntaxWarning"
+
+# openstack-rate-limit
+rate_limit:
+  enabled: false
+  rate_limit_by: target_project_id
+  max_sleep_time_seconds: 0
+  log_sleep_time_seconds: 10
+  backend_timeout_seconds: 1
+  whitelist:
+    - Default/service
+    - Default/admin
+    - tempest/performance-testing
+    - ccadmin/cloud_admin
+    - ccadmin/master
+  whitelist_users: null
+  blacklist: null
+  blacklist_users: null
+  groups:
+    write:
+      - delete
+      - update
+  rates:
+    default:
+      servers:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      servers/server:
+        - action: read/list
+          limit: 100r/m
+        - action: write
+          limit: 100r/m
+      servers/*/migrations:
+        - action: read/list
+          limit: 100r/m
+      servers/*/migrations/migration:
+        - action: read/list
+          limit: 100r/m
+      servers/*/action:
+        - action: create
+          limit: 100r/m
+      flavors:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      flavors/flavor:
+        - action: read/list
+          limit: 100r/m
+        - action: write
+          limit: 100r/m
+      os-keypairs:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      os-aggregates:
+        - action: read/list
+          limit: 100r/m
+        - action: create
+          limit: 100r/m
+      os-aggregates/os-aggregate:
+        - action: read/list
+          limit: 100r/m
+        - action: write
+          limit: 100r/m
+      os-availability-zone:
+        - action: read/list
+          limit: 100r/m
+      os-hypervisors:
+        - action: read/list
+          limit: 100r/m
+      os-server-groups:
+        - action: read/list
+          limit: 100r/m
+        - action: write
+          limit: 100r/m
+      os-server-groups/os-server-group:
+        - action: read/list
+          limit: 100r/m
```

Comment:
These are _all_ 100r/m. Did you check which numbers may make sense (perhaps going by current peak usage), or is this left for the future?